docs/DATA_FORMATS_INPUT.md

```diff
@@ -272,3 +272,156 @@ There are many more options available,
     "measurement*"
   ]
 ```
+
+## LTSV:
+
+The [Labeled Tab-separated Values (LTSV)](http://ltsv.org/) data format translate a LTSV line into a measurement with _timestamp_, _fields_ and _tags_. For example, this line:
+
+```
+time:2016-03-06T09:24:12Z\tstr1:value1\tint1:23\tint2:34\tfloat1:1.23\tbool1:true\tbool2:false\tignore_field1:foo\ttag1:tval1\tignore_tag1:bar\ttag2:tval2
+```
+
+Would get translate into _timestamp_, _fields_ and _tags_ of a measurement using the example configuration in the following section:
+
+```
+ltsv_example str1=value1,int1=23i,int2=34i,float1=1.23,bool1=true,bool2=false tag1=tval1,tag2=tval2,log_host=log.example.com 2016-03-06T09:24:12Z
+```
+
+### LTSV Configuration:
+
+The LTSV data format specifying the following configurations.
+
+- metric_name
+- time_label
+- time_format
+- str_field_labels
+- int_field_labels
+- float_field_labels
+- bool_field_labels
+- tag_labels
+- duplicate_points_modifier_method
+- duplicate_points_modifier_uniq_tag
+
+For details, please see the comments in the following configuration example.
+
+```toml
+[[inputs.tail]]
+  ## The measurement name
+  override_name = "nginx_access"
+
+  ## A LTSV formatted log file path.
+  ## See http://ltsv.org/ for Labeled Tab-separated Values (LTSV)
+  ## Here is an example config for nginx (http://nginx.org/en/).
+  ##
+  ##  log_format  ltsv  'time:$time_iso8601\t'
+  ##                    'host:$host\t'
+  ##                    'http_host:$http_host\t'
+  ##                    'scheme:$scheme\t'
+  ##                    'remote_addr:$remote_addr\t'
+  ##                    'remote_user:$remote_user\t'
+  ##                    'request:$request\t'
+  ##                    'status:$status\t'
+  ##                    'body_bytes_sent:$body_bytes_sent\t'
+  ##                    'http_referer:$http_referer\t'
+  ##                    'http_user_agent:$http_user_agent\t'
+  ##                    'http_x_forwarded_for:$http_x_forwarded_for\t'
+  ##                    'request_time:$request_time';
+  ##  access_log  /var/log/nginx/access.ltsv.log  ltsv;
+  ##
+  filename = "/var/log/nginx/access.ltsv.log"
+
+  ## Seek to this location before tailing
+  seek_offset = 0
```

Comment:
do you think that is a useful option to expose to all users? How would users determine the correct byte offset that they want to seek to? What happens if they specify wrong?